stdlib/public/SwiftShims/swift/shims/LibcShims.h

```diff
@@ -61,7 +61,7 @@ SWIFT_READONLY
 static inline int _swift_stdlib_memcmp(const void *s1, const void *s2,
                                        __swift_size_t n) {
 // FIXME: Is there a way to identify Glibc specifically?
-#if defined(__gnu_linux__)
+#if defined(__gnu_linux__) || defined(__ANDROID__)
```

Comment:
This is the only line that will break the last NDK 25c, as there's no way to check the NDK version without including headers like `sys/cdefs.h`, which this header doesn't include.